src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const obj = { ...state };
+  const arr = [];
+
+  for (const char of actions) {
```

Comment:
what is char?)
Google: <img width="558" alt="image" src="https://user-images.githubusercontent.com/86183007/190343598-59c84155-f2c0-4f55-882a-f29534b20d44.png">